docs/site/Concepts.md

```diff
@@ -6,7 +6,9 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Concepts.html
 ---
 
-LoopBack 4 introduces some new concepts that are important to understand:
+LoopBack 4 introduces some new concepts that are important to understand.
+
+![Key concepts overview diagram](imgs/key-concepts-overview-diagram.png)
 
 - [**Application**](Application.md): The central class for setting up all of
   your module’s components, controllers, servers and bindings. The Application
```

Comment:
We may need to refer to a lower layer that shows `context`, `binding`, and `component`.

Reply:
LoopBack 4 defines some key building blocks to represent different responsibilities for typical API and/or Microservice applications.

Reply:
> We may need to refer to a lower layer that shows context, binding, and component.

@raymondfeng , do you want to move those to the "behind the scene" sidebar?

Reply:
Maybe.

Reply:
Should we move "interceptor" to section "Behind-the-scene" as well?

Reply:
Interceptor probably should stay outside of Behind-the-scene.